src/js/model/primitives/marks/Group.test.js

```diff
@@ -0,0 +1,104 @@
+/* eslint no-unused-expressions:0 */
+'use strict';
+
+var expect = require('chai').expect;
+
+var Group = require('./Group');
+var Mark = require('./Mark');
+
+describe('Group Mark', function() {
+  var group;
+
+  beforeEach(function() {
+    group = new Group();
+  });
+
+  describe('constructor', function() {
+
+    it('is a constructor function', function() {
+      expect(Group).to.be.a('function');
+    });
+
+    it('may be used to create group instances', function() {
+      expect(group).to.be.an.instanceOf(Group);
+    });
+
+    it('inherits from Mark', function() {
+      expect(group).to.be.an.instanceOf(Mark);
+    });
+
+  });
+
+  describe('default properties', function() {
+
+    it('contains a properties object', function() {
+      expect(group).to.have.property('properties');
+    });
+
+    it('is initialized with a scales array', function() {
+      expect(group).to.have.property('scales');
+      expect(group.scales).to.deep.equal([]);
+    });
+
+    it('is initialized with a legends array', function() {
+      expect(group).to.have.property('legends');
+      expect(group.legends).to.deep.equal([]);
+    });
+
+    it('is initialized with a axes array', function() {
+      expect(group).to.have.property('axes');
+      expect(group.axes).to.deep.equal([]);
+    });
+
+    it('is initialized with a marks array', function() {
+      expect(group).to.have.property('marks');
+      expect(group.marks).to.deep.equal([]);
+    });
+
+  });
+
+  describe('child method', function() {
+
+    it('is a function', function() {
+      expect(group).to.have.property('child');
+    });
+
+    it('is a function', function() {
```

Comment:
Good call. I wanted the first to be `it('exists'`. I'm used to breaking these up to keep my `it` blocks as granular as possible; if you'd prefer I combine them I will do so.